docs/dev-manuals/architecture/domain-tradeoff-analysis.md

```diff
@@ -0,0 +1,163 @@
+
+## Product
+### Usability Requirements
+#### R-U-001: As realistic as possible
+Both domains allow for a realistic system, since similar - microservice-based - systems are in use today. The SA system might be more relevant after the project for the chair since it fits better to their research focus. Since the SCS system is built on top of a simulation, it is slightly less realistic. Since container ships are quite slow, a time speedup is needed.
+The TS system will also need simulated "processing" of the tickets.
+
+#### R-U-003: Configurations
+Configuration is essential for the experiment-focused side of the reference architecture. The requirement R-U-003 also includes architecture-side configuration and different variants of the system which are indifferent between the domains. The following comparison focuses on the deployment configuration.
+There are more configurable variables in the SCS domain since all shipping-related variables (like boat size, container size, number of boats, number of ports, frequency of location updates), that can be adjusted based on the researcher's needs.
+
+#### R-U-004: Experiments
+While the SCS domain is more configurable (see R-U-003), it is more effort to make experiments deterministic. Since the system is stateful due to the current location of ships and containers, a well-defined starting state would need to be introduced.
+
+#### R-U-005: The RA must be *lightweight*
+Due to the reduced technical complexity of the TS domain, it can be considered the most lightweight.
+
+#### R-U-008: Observability
+The SCS domain might be more complicated to observe since actions and events are less user-input-driven than in the SA domain. For example, the metrics regarding one pipeline call can easily be measured and observed end-to-end, while a shipping request adds the order in the shipping queue and leads to a lot of follow-up calculations and optimizations, that are harder to link to one request.
+
+#### R-U-011: Adding services
+Due to the looser coupling of the SA domain, additional microservices should be easier to add as long as they do not influence the core services (like scheduling/pipelines).
+
+#### R-U-012: True to the term *Reference Architecture*
+See R-U-005: Since the TS system will be more lightweight and easy to understand, it has a slight edge over the other domains.
+
+#### R-U-013: Deterioration in performance
+Both the SA and SCS domains allow for performance deterioration, although through different means. In the SA domain, heavy load can be put on the system by adding AI-driven steps. In the SCS domain, the routing and optimization for large orders and ships or data-intensive business intelligence queries can add load.
+The TS system does not include similar - resource-intensive processes.
+
+### ToAdd Unexpected load
```

Comment:
Is that intended?

Reply:
Yes, the issue is that this is a requirement, that was so far not tracked in the official requirements file. Therefore, it does not yet have a identifier